app/assets/stylesheets/budget_style.css

```diff
@@ -25,6 +25,7 @@ body {
   align-items: center;
   height: 90vh;
   font-weight: 800;
+  font-family: "Nova script", sans-serif;
 }
 
```

Comment:
- [ ] Hey there! 👋 Awesome progress so far! 🚀 Could you please take a look at the styling? 🎨 In the current setup, the category icon is overlapping, and that might not provide the best user experience. 🤔 Let's make sure everything looks clean and organized! Thanks! 🙌

![image](https://github.com/ismayito/Rails-budget-app/assets/112261237/79fcea02-fa89-4d8e-bac1-a312e9857e7d)